recipes/pybip39/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "pybip39" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/kevinheavey/pybip39/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 79738cab062711634e8a34e3a37c4b50cdb8ba0704f6f99aa4c5e29cd409afce
+  patches:
+    # Upstream uses git version pyo3=0.15.1 that defaults to 0.15.2 and fails
+    # to build. This patch updates the version to pyo3=0.16.6.
+    - update-Cargo.toml-pyo3-version.patch
+
+build:
+  script:
+    - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
+    - {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('rust') }}
+    - cargo-bundle-licenses
+    - python
```

Comment:
This should not be here and will likely lead to break, rather try setting `PYO3_PYTHON` for windows.
```suggestion
```

Reply:
It passes though :'(

Do you think that this should work:
```
  script_env:
    - PYO3_PYTHON: %PREFIX%\bin\python.exe  # [win]
```
or:
```
  script_env:
    - PYO3_PYTHON: {{ PYTHON }}  # [win]
```
Well, the latter should work

Reply:
Yes, try that. The other one might lead to an empty package at one point.